cmds.go

```diff
@@ -224,6 +224,34 @@ func exports(socket, certPath string) map[string]string {
 		out["DOCKER_TLS_VERIFY"] = "1"
 	}
 
+	//if a http_proxy is set, we need to make sure the boot2docker ip
+	//is added to the NO_PROXY environment variable
+	if os.Getenv("http_proxy") != "" || os.Getenv("HTTP_PROXY") != "" {
+		//get the ip from the docket/DOCKER_HOST
+		re := regexp.MustCompile("tcp://([^:]+):")
+		if matches := re.FindStringSubmatch(socket); len(matches) == 2 {
+			ip := matches[1]
+
+			//first check for an existing lower case no_proxy var
+			no_proxy_var := "no_proxy"
+			no_proxy_value := os.Getenv("no_proxy")
+			//otherweise try allcaps HTTP_PROXY
```

Comment:
s/otherweise/otherwise/
```go
// otherwise try all caps
```
or just git rid of the comment altogether, as it is pretty obvious and doesn't need a comment, in my opinion.
In fact, you could probably get rid of the comment on line 235 as well.